modules/virt-installing-virtctl-client-web.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// install/virt-installing-virtctl.adoc
+
+[id="virt-installing-virtctl-client-web_{context}"]
+= Installing the `virtctl` client from the web console
+
+You can download the `virtctl` client from the Red Hat Customer Portal, which is linked to in your {VirtProductName} web console in the *Command Line Tools* page.
+
+.Prerequisites
+
+* You must have an activated {product-title} subscription to access the download page on the Customer Portal. 
+
+.Procedure
+
+. Access the Customer Portal by clicking the image:../images/question-circle.png[title="Help"] icon, which is in the upper-right corner of the web console, and selecting *Command Line Tools*. 
```

Comment:
While this technically does work just fine, since the `images/` symlinks are in place, this should be able to be simplified as `image:question-circle.png[title="Help"]`, as shown in:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#inserting-images-inline-without-reuse

Reply:
Ah, TIL. Thank you :)